sdk/cosmosdb/cosmos/src/client/Container/Containers.ts

```diff
@@ -104,7 +104,7 @@ export class Containers {
     body: ContainerRequest,
     options: RequestOptions = {}
   ): Promise<ContainerResponse> {
-    const err = {};
+    const err: { message?: string } = {};
```

Comment:
Kind of out of scope for this PR, but this is not ideal. We should throw proper instances of error"

throw new Error(message)

Would that be easy to clean up here?

Reply:
I'll clean up the relevant scope, but doesn't seem trivial to clean up the helper right now. Would be another PR